docs/source/Schemas.md

```diff
@@ -189,6 +189,10 @@ additionally a field with the suffix `_type` is generated that holds
 the corresponding enum value, allowing you to know which type to cast
 to at runtime.
 
+Union do not support scalar data types. As a workaround, you can wrap a scalar
+value in a table or in a struct. The latter is more space-efficient but lacks
+the schema evolution features of a table.
+
```

Comment:
Since this text doesn't even talk about structs, a better fix would be to change the last paragraph:

`There is an experimental support only in C++ for a vector of unions (and types). In the example IDL file above, use [Any] to add a vector of Any to Monster table. There is also experimental support for other types besides tables in unions, in particular structs and strings. There's no direct support for scalars in unions, but they can be wrapped in a struct at no cost.`

Reply:
PR updated! I didn't realise that structs in unions was an experimental feature, and I wasn't aware of the possibility of unions of strings. Thanks for the clarifications!